hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/autoReloadPanel/autoReloadPanel.tsx

```diff
@@ -77,7 +77,7 @@ class AutoReloadPanel extends React.Component<IAutoReloadPanelProps> {
      const lastUpdatedDeltaFullText = lastUpdatedOMDBDelta === 0 || lastUpdatedOMDBDelta === undefined || lastUpdatedOMDBFull === 0 || lastUpdatedOMDBFull === undefined ? '' :
      (
       <>
-      &nbsp; | OM DB updated at {lastUpdatedDeltaFullToolTip}
+      &nbsp; | DB Sync updated at {lastUpdatedDeltaFullToolTip}
```

Comment:
```suggestion
      &nbsp; | DB Synced at {lastUpdatedDeltaFullToolTip}
```
`DB Sync updated` sounds different.

Reply:
Completed Changes Please review.